pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/route/UserCreationRoute.java

```diff
@@ -47,29 +49,8 @@ public Object handle(Request request, Response response, UserCreationPayload pay
         */ 
         final var requestorClientId = auth.getClient();
         final var domain = auth.getDomain();
-        final var studyGuid = payload.getStudyGuid();
-
-        final var email = payload.getEmail();
-        final var emailValidator = EmailValidator.getInstance();
-
-        /* 
-        * It's not absolutely necessary to check here- the ParticipantsCreateService will also
-        *  check that the email is correctly formatted (you'd need to update its associated
-        *  switch statement below in the transaction)
-        *
-        * Checking here is in order to save a few cycles & connections to the DB if it's
-        *  immediately clear the client gave us bad data.
-        */
-        if (emailValidator.isValid(email) == false) {
-            var error = new ApiError(ErrorCodes.MALFORMED_EMAIL, "the email address is not in a valid format.");
-            throw ResponseUtil.haltError(HttpStatus.SC_UNPROCESSABLE_ENTITY, error);
-        }
-
-        if (StringUtils.isNotBlank(payload.getCenterId())) {
-            var error = new ApiError(ErrorCodes.NOT_SUPPORTED,
-                    "passing a non-null center id when creating a participant is not supported yet.");
-            throw ResponseUtil.haltError(HttpStatus.SC_UNPROCESSABLE_ENTITY, error);
-        }
+        final var studyGuid = new Guid(payload.getStudyGuid());
+        final var email = new EmailAddress(payload.getEmail());
```

Comment:
My intent here is to bake the validation done on the email into the type-  Instead of implicitly assuming that the content of the string has been validated further up the call-tree, an object of this type can only be created if the email address is correctly formatted, and that validation is explicitly declared through the type.

This can be trivially removed if folks feel this is over-engineered.